recipe/meta.yaml

```diff
@@ -24,57 +27,64 @@ requirements:
     - pip
     - setuptools
     - wheel
+    - packaging
   run:
    - python
-   - applaunchservices >=0.3.0  # [osx]
+   - python.app                        # [osx]
+   # This is here to work around a bug in mamba
+   - ptyprocess >=0.5                  # [win]
+   - aiohttp >=3.9.3
+   - applaunchservices >=0.3.0         # [osx]
+   - asyncssh >=2.14.0,<3.0.0
    - atomicwrites >=1.2.0
    - chardet >=2.0.0
    - cloudpickle >=0.5.0
    - cookiecutter >=1.6.0
    - diff-match-patch >=20181111
+   - importlib-metadata >=4.6.0        # [py<3.10]
    - intervaltree >=3.0.2
-   - ipython >=8.12.2,<8.13.0  # [py==38]
+   - ipython >=8.12.2,<8.13.0          # [py==38]
    - ipython >=8.13.0,<9.0.0,!=8.17.1  # [py>38]
    - jedi >=0.17.2,<0.20.0
-   - jellyfish >=0.7          # [not (osx and arm64 and py<312)]
+   - jellyfish >=0.7                   # [not (osx and arm64 and py<312)]
    # Tests fail with "jellyfish 1.0.1 is not supported on this platform"
-   - jellyfish >=0.7,!=1.0.1  # [osx and arm64 and py<312]
+   - jellyfish >=0.7,!=1.0.1           # [osx and arm64 and py<312]
    - jsonschema >=3.2.0
    - keyring >=17.0.0
    - nbconvert >=4.0
    - numpydoc >=0.6.0
-   - paramiko >=2.4.0  # [win]
    - parso >=0.7.0,<0.9.0
    - pexpect >=4.4.0
    - pickleshare >=0.4
-   # This is here to work around a bug in mamba
-   - ptyprocess >=0.5  # [win]
    - psutil >=5.3
+   - pygithub >=2.3.0
    - pygments >=2.0
-   - pylint >=2.5.0,<3.1
+   - pylint >=3.1,<4
    - pylint-venv >=3.0.2
-   - python-lsp-black >=2.0.0,<3.0.0
    - pyls-spyder >=0.4.0
-   - pyqt >=5.10,<5.16
-   - pyqtwebengine >=5.10,<5.16
-   - python.app  # [osx]
-   - python-lsp-server >=1.10.0,<1.11.0
-   - pyxdg >=0.26  # [linux]
-   - pyzmq >=22.1.0
+   - pyqt >=5.15,<5.16
+   - pyqtwebengine >=5.15,<5.16
+   - python-lsp-black >=2.0.0,<3.0.0
+   - python-lsp-server >=1.12.0,<1.13.0
```

Comment:
Spyder vendored python-lsp-server here https://github.com/spyder-ide/spyder/tree/v6.0.1/external-deps/python-lsp-server, but we provide our `python-lsp-server`. 
The maintainers expect _python-lsp-server[**all**]>=1.12.0,<1.13.0_ https://github.com/spyder-ide/spyder/blob/v6.0.1/setup.py#L238. 
That means additional linters and formatters https://github.com/python-lsp/python-lsp-server/blob/v1.12.0/pyproject.toml#L29-L40. 
Do we need them, or can we skip them? Because we haven't added them before.

BTW, we have all those packages on the `defaults` channel.

Reply:
Yeah, I'm fine with your solution